src/sentry/static/sentry/app/locale.jsx

```diff
@@ -178,15 +187,15 @@ function mark(rv) {
     key: null,
     ref: null,
     props: {
-      className: 'translation-wrapper',
+      className: markerCss,
       children: _.isArray(rv) ? rv : [rv],
     },
     _owner: null,
     _store: {},
   };
 
   proxy.toString = function() {
-    return '🇦🇹' + rv + '🇦🇹';
+    return '✅' + rv + '✅';
```

Comment:
The red seemed to imply a bit too much 'missing translation' to me.

Reply:
Rip Austria

Reply:
oh it's a flag lol, I thought it was like, the NO symbol.

![image](https://user-images.githubusercontent.com/1421724/53138036-ce2d6400-3539-11e9-96ed-47362723c086.png)

I think my emoji font makes it unclear.